core/src/test/java/org/apache/calcite/test/JdbcAdapterTest.java

```diff
@@ -1355,6 +1355,27 @@ private LockWrapper exclusiveCleanDb(Connection c) throws SQLException {
         .runs();
   }
 
+  /**
+   * Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6995">[CALCITE-6995]
+   * StarRocks/Doris should support full join</a>.*/
+  @Test void testFullJoinSupportedDialect() {
+    CalciteAssert.model(JdbcTest.SCOTT_MODEL)
+        .enable(CalciteAssert.DB != CalciteAssert.DatabaseInstance.H2
```

Comment:
Q: I see no other tests where we enable when the DB is not set to a given set of values (inequality), we always enable on specific values (equality). In this case, do we want to enable it specifically for Doris/StarRocks or for all DBs except the two you have excluded?

Reply:
I think here should test for all DBs except the two excluded, because other db can support full join. @asolimando